lighthouse/environment/src/lib.rs

```diff
@@ -27,6 +23,16 @@ use task_executor::{ShutdownReason, TaskExecutor};
 use tokio::runtime::{Builder as RuntimeBuilder, Runtime};
 use types::{EthSpec, MainnetEthSpec, MinimalEthSpec};
 
+#[cfg(target_os = "linux")]
```

Comment:
You may be able to relax this from `"linux"` to `"unix"` since thats the os targe tokio_signal uses. Also, it looks like there is windows support in tokio_signal: https://github.com/tokio-rs/tokio/blob/master/tokio/src/signal/windows.rs

Reply:
I went ahead and relaxed the cfg from `linux` to `unix`!

I took a look into using the windows support in tokio::signal but it looks like that is all private to tokio that they don't allow consuming code to access. The only thing that is exposed that has the windows support is the `ctrl_c` function that returns an `io::Result<()>`. So while the functions seem to be there for windows, I don't think it's something we can access at this point.